File: sdk/ObservationImportant+CoreDataProperties.m
//
//  ObservationImportant+CoreDataProperties.m
//  mage-ios-sdk
//
//  Created by William Newman on 9/20/16.
//  Copyright © 2016 National Geospatial-Intelligence Agency. All rights reserved.
//

#import "ObservationImportant+CoreDataProperties.h"

@implementation ObservationImportant (CoreDataProperties)

+ (NSFetchRequest<ObservationImportant *> *)fetchRequest {
	return [[NSFetchRequest alloc] initWithEntityName:@"ObservationImportant"];
}

@dynamic dirty;
@dynamic important;
@dynamic timestamp;
@dynamic userId;
@dynamic reason;
@dynamic observation;

@end

File: Mage/CoreData/ObservationImportant+CoreDataProperties.swift
//
//  ObservationImportant+CoreDataProperties.m
//  mage-ios-sdk
//
//  Created by William Newman on 9/20/16.
//  Copyright © 2016 National Geospatial-Intelligence Agency. All rights reserved.
//

import Foundation
import CoreData

extension ObservationImportant {
    @nonobjc public class func fetchRequest() -> NSFetchRequest<ObservationImportant> {
        return NSFetchRequest<ObservationImportant>(entityName: "ObservationImportant")
    }
    
    @NSManaged var dirty: Bool
    @NSManaged var important: Bool
    @NSManaged var timestamp: Date?
    @NSManaged var reason: String?
    @NSManaged var userId: String?
    @NSManaged var observation: Observation?
    
}
